openslides/core/static/js/core/site.js

```diff
@@ -354,6 +360,32 @@ angular.module('OpenSlidesApp.core.site', [
     }
 ])
 
+// Options for TinyMCE editor used in various create and edit views.
+.factory('EditorOptions', [
+    'gettextCatalog',
+    'Mediafile',
```

Comment:
I need Mediafile here to use 'EditorOptions' as generic factory. The alternative: I have to duplicate EditorOptions in all apps where tinymce is used (core, motions, users).
And there is already Mediafile used in core (see CustomslideForm).
So wan't fix.